ios/Podfile.lock

```diff
@@ -339,17 +360,18 @@ EXTERNAL SOURCES:
 
 SPEC CHECKSUMS:
   Amplitude-iOS: df355e2be58972041acde9576a2ae97c217bf941
-  AppCenter: 513d32888854d67d8cfbd3a8db16aeb5fb2e2a75
+  AppCenter: ca66175050d538b157959382dd43f1ab96cdab84
   appcenter: f2e241f931e0865a74b3af3928066a1190814be0
   appcenter-analytics: 42e368870a904af67fd93f043f3efa75ffa772a2
   appcenter-crashes: bfc5c03a6ddcfa158fcf4217d1724fd52f45d12c
-  AppCenterReactNativeShared: c4eb2ad593c32fbc2fc5f2ea9f5cf1faae61c182
+  AppCenterReactNativeShared: d1218878427a69d4b76f73883256455ed20f12b3
```

Comment:
Should I be worried that these checksums changed? I don't remember them changing in the past.

Reply:
I can understand that there would be additions for new things but not sure it makes sense for old things?

Reply:
this file is generated, not merged